internal/cmdplan/command.go

```diff
@@ -265,24 +264,82 @@ func printWithPrefix(text, prefix string, ioStreams genericclioptions.IOStreams)
 
 // printKRM outputs the plan inside a ResourceList so the output format
 // follows the KRM function wire format.
-func printKRM(plan *v1alpha1.Plan, ioStreams genericclioptions.IOStreams) error {
-	b, err := yaml.Marshal(plan)
+func printKRM(
+	plan *kptplanner.Plan,
+	ioStreams genericclioptions.IOStreams,
+) error {
+	planResource, err := yaml.Parse(strings.TrimSpace(`
+apiVersion: kpt.dev/v1alpha1
+kind: Plan
+metadata:
+  name: plan
+  annotations:
+    config.kubernetes.io/local-config: true	
+`))
 	if err != nil {
-		return fmt.Errorf("failed to marshall plan into yaml: %w", err)
+		return fmt.Errorf("unable to create yaml document: %w", err)
 	}
-	rn, err := yaml.Parse(string(b))
+
+	sNode, err := planResource.Pipe(yaml.LookupCreate(yaml.SequenceNode, "spec", "actions"))
```

Comment:
Q: Why not use "normal" marshalling?  Is it because we want to preserve comments in the yaml?  Can we add a (code) comment?